foodhub_custom/models/foodhub_product.py

```diff
@@ -0,0 +1,7 @@
+from odoo import fields, models
```

Comment:
We usually name the files following the model name. Here, that would be `product.py`.

Reply:
Done !